### ISSUE TITLE
The problem of reproducing the accuracy of the paper

### ISSUE THREAD

**Issue by BigBigBigBigPotato** [ChenHongruixuan/ChangeMamba#104]
Unable to reproduce the accuracy of MambaSCD-Small on SECOND dataset. The best Sek is 21.9，which cannot be achieved 22.83 in paper，no need to say 24.2 in "code with paper".
Every training session will have the following errors,if this influence the final accuracy?

Successfully load ckpt /Project/MambaCD/pretrained_weight/vssm_small_0229_ckpt_epoch_222.pth
_IncompatibleKeys(missing_keys=['outnorm0.weight', 'outnorm0.bias', 'outnorm1.weight', 'outnorm1.bias', 'outnorm2.weight', 'outnorm2.bias', 'outnorm3.weight', 'outnorm3.bias'], unexpected_keys=['classifier.norm.weight', 'classifier.norm.bias', 'classifier.head.weight', 'classifier.head.bias'])

**Comment by AnshSG13** [comment]
I have the same error message
